docs/CHANGELOG.html

```diff
@@ -44,12 +44,12 @@
 
 
 
-      <h1 data-sassdoc="project-title">
+      <div data-sassdoc="project-title">
         <a href="index.html" rel="home" data-sassdoc="project-name">Herman</a>
```

Comment:
Looks odd to me that the link underlines are pale blue (though I do like that the sidebar nav underlines are pale blue). I very much like that the underlines are not bold and obtrusive. What would it look like if the underlines were a very faded version of the red link color? I like the idea of hiding underlines for headings.
Also it's a little odd that links like @font and @preview are blue with a blue underline and then just the underline turns pink. Another thing that happened is I found myself thinking that the blue without underlines were links too. Basically, this whole link situation feels a bit chaotic in terms of color.

Reply:
Here's a lighter red underline:

![link screenshot](https://dl.dropbox.com/s/1d6b9dm3uay2gqk/Screenshot%202017-11-22%2014.38.24.jpg)

The blue isn't about links at all, that's the color we've been using for bits of inline code. When code is also linked, it seems like the code color is taking precedence - which we could fix. If the code style also looks like a link style to you, we should come up with a different way to make code stand out.

Reply:
Here's an attempt at using bold instead of blue for inline code. Too heavy handed?

![bold code screenshot](https://dl.dropbox.com/s/xarg4om42dh7p1r/Screenshot%202017-11-22%2014.52.41.jpg)

Reply:
(I switched to a semibold for code. pull the latest to see it)